labrad_server/bat-script/set_env_variable.bat

```diff
@@ -0,0 +1,20 @@
+@echo off
+
+set /p choose= Input your IP, and press "Enter"
+if "%choose%"=="" (
```

Comment:
The batchfile helps to add some environmental variables. 

Labradhost can be localhost, or other accessible IP address which enable remote control.